pom.xml

```diff
@@ -31,7 +31,6 @@
     <version.junit>4.13.2</version.junit>
     <version.assertj>3.20.2</version.assertj>
     <version.jackson>2.13.1</version.jackson>
```

Comment:
Feel free to create a follow-up ticket but it should be nice to updat Jackson version to stay aligned with Drools ( https://github.com/kiegroup/drools/blob/main/build-parent/pom.xml#L43C74-L43C74 ).
Actually it could be interesting to inherit the version from drools pom directly.
Wdyt? Is it fine for you to create a follow up ticket?

Reply:
will do.

Reply:
see https://github.com/kiegroup/drools-ansible-rulebook-integration/pull/62